examples/code-highlighting/index.js

```diff
@@ -239,6 +263,55 @@ class CodeHighlighting extends React.Component {
 
     return [...others, ...decorations]
   }
+
+  /**
+   * On paste inside code blocks, insert code lines . Fix the original error of pasting html content.
+   * Note that: paste any unknown html, fragment content should not be supported in code blocks
+   *
+   * @param {Event} event
+   * @param {Editor} editor
+   * @param {Function} next
+   */
+
```

Comment:
I'm unsure whether we need to have a paste handler in this example. These examples are not meant to be fully featured.

Reply:
@CameronAckermanSEL 
I agree with you, the examples are not meant to be fully featured. However, it will cause a crash when pasting. Official examples may lack some features, but should not report any fatal errors in any way while the user is performing normal operations.
I still hope that the official example code is robust and worthy of reference.

Reply:
@CameronAckermanSEL @CodePlayer A paste handler in the example seems reasonable since (a) it's minimum viable feature if it crashes and (b) It is separate from the syntax highlight part of the code, so it doesn't add complexity to understanding it. If unneeded, easy to ignore.